source/borderrouter_tasklet.c

```diff
@@ -59,6 +55,9 @@ conf_t *global_config = static_config;
 #define NR_BACKHAUL_INTERFACE_PHY_DRIVER_READY 2
 #define NR_BACKHAUL_INTERFACE_PHY_DOWN  3
 
+static mac_api_t *api;
```

Comment:
Later on, you assume *api to default to `NULL`
Please initialize this to `NULL` just to be safe.